### PR TITLE
[DEV-605, DEV-607, DEV-267] - Add guides lists

### PR DESCRIPTION
**Pull request by marcobottaro**
Adds pagoPA, send and ioSign guides and manuals pages

#### List of Changes
Add layout to guides page and create responsive card components

#### How Has This Been Tested?
Manually

#### Screenshots (if appropriate):
image

#### Types of changes
- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/_contents/products.ts [hunk 1]
```diff
@@ -4,12 +4,12 @@ import { pagoPaOverview } from '@/_contents/pagoPa/overview';
...
 import { ioSign } from './ioSign/ioSign';
-import { pagoPA } from './pagoPa/pagoPA';
+import { pagoPa } from './pagoPa/pagoPa';
 
 export const overviews = [appIoOverview, ioSignOverview, pagoPaOverview];
 export const quickStarts = [];
 export const apis = [];
 export const tutorials = [];
 export const tools = [];
 export const productsGuides = [pagoPaGuides];
```
Comment (datalek) [inline apps/nextjs-website/src/_contents/products.ts:14]: The `productsGuides` is going to be ambiguous once the gitbook guides are integrated. What do you think to rename it to `guideHomepages`?

File: apps/nextjs-website/src/components/atoms/NavigateButton/LinkButton.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+export type LinkButtonProps = {
+  readonly href: string;
+  readonly icon?: string;
+  readonly title: string;
+};
+
+export const LinkButton: FC<LinkButtonProps> = ({
+  href = '',
+  title = '',
```
Comment (datalek) [inline apps/nextjs-website/src/components/atoms/NavigateButton/LinkButton.tsx:13]:
```suggestion
  href,
  title,
```

File: apps/nextjs-website/src/lib/api.ts [hunk 3]
```diff
@@ -21,6 +21,6 @@ export function getGuides(productSlug?: string) {
 
 export function getOverview(productSlug?: string) {
   return overviews.find(
-    (overviewData) => overviewData.product.path === `/${productSlug}`
+    (overviewData) => overviewData.product?.path === `/${productSlug}`
```
Comment (datalek) [inline apps/nextjs-website/src/lib/api.ts:24]:
```suggestion
    (overviewData) => overviewData.product.path === `/${productSlug}`
```

File: apps/nextjs-website/src/lib/types/guidesData.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
 import { Page } from '@/lib/types/page';
 import { GuidesPageProps } from '@/pages/[productSlug]/guides';
 
-export type GuidesData = Page<GuidesPageProps>;
+export type GuidesData = Page<Omit<GuidesPageProps, 'products'>>;
```
Comment (datalek) [inline apps/nextjs-website/src/lib/types/guidesData.ts:4]: The property `products` does not exist
```suggestion
export type GuidesData = Page<GuidesPageProps>;
```

File: apps/nextjs-website/src/pages/[productSlug]/guides.tsx [hunk 5]
```diff
@@ -14,27 +20,56 @@ export const getStaticPaths: GetStaticPaths<Params> = () => ({
...
+  path,
+}: GuidesPageProps) => {
+  const { palette } = useTheme();
+
+  return (
+    <Layout products={products} product={product} path={path}>
+      <>
+        {/*<Breadcrumbs size="md">
+          <Link />
+        </Breadcrumbs>*/}
```
Comment (datalek) [inline apps/nextjs-website/src/pages/[productSlug]/guides.tsx:55]:
```suggestion
```

**Review by datalek — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/lib/types/guidesData.ts (hunk 4 above)
Comment (datalek) [reply apps/nextjs-website/src/lib/types/guidesData.ts:4]: Ignore this suggestion. I didn't see that `LayoutProps` has the `products` property.

File: apps/nextjs-website/src/components/atoms/GuideCard/GuideCard.tsx
Comment (kin0992) [inline apps/nextjs-website/src/components/atoms/GuideCard/GuideCard.tsx:1]: I guess this component should be a inside the molecules folder: it imports 2 atoms, so I guess it _cannot be_ an atom. am I right?

File: apps/nextjs-website/src/components/atoms/GuideCard/GuideCard.tsx [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+        <Box sx={{ display: 'flex', flexDirection: 'column', width: '100%' }}>
+          <CardContent>
+            <Typography variant='h6' mb={spacing(1.75)}>
+              {title}
+            </Typography>
+            <Typography variant='subtitle2' color='text.primary'>
+              {description.title}
+            </Typography>
+            <UnorderedList listItems={description?.listItems} />
+            <LinkButton {...link}></LinkButton>
```
Comment (kin0992) [inline apps/nextjs-website/src/components/atoms/GuideCard/GuideCard.tsx:71]:
```suggestion
            <LinkButton {...link} />
```

**Review by kin0992 — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/components/atoms/GuideCard/GuideCard.tsx
Comment (marcobottaro) [reply apps/nextjs-website/src/components/atoms/GuideCard/GuideCard.tsx:1]: Good point! I moved it

**Review by kin0992 — COMMENTED** [review]
I think appIO guide index page is missing

**Review by kin0992 — APPROVED** [review]
👍

**Review by datalek — APPROVED** [review]
📖

**Pull request merged**